x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/timeline/_timeline.scss

```
@@ -0,0 +1,77 @@
+$colorBarHeight: $euiSizeS;
```

```suggestion
$ilmTimelineBarHeight: $euiSizeS;
```

Just in case, this will be sure it stays scoped to these usages.